docker/isotovideo/Dockerfile.qemu-x86

```diff
@@ -0,0 +1,5 @@
+#!BuildTag: isotovideo:qemu-x86
+
+FROM opensuse/tumbleweed
+RUN zypper -n in os-autoinst qemu-x86 qemu-tools
```

Comment:
Could you please clarify why the qemu-x86 qemu-tools packages has to be added?

Reply:
yes. os-autoinst supports multiple backends. qemu is just one of them so this is why I built multiple variants in https://build.opensuse.org/project/show/home:okurz:container, e.g. isotovideo-qemu, isotovideo-qemu-kvm, isotovideo-qemu-x86, see the build tag and the filename. Do you think we can better specify that specific variant elsewhere or differently? We could add specific subpackages to https://github.com/os-autoinst/os-autoinst/blob/master/dist/rpm/os-autoinst.spec#L92 and install these.

Reply:
Thanks for the explanation. Could it be possible to do it architecture independent as we did on https://github.com/os-autoinst/openQA/blob/6a2b837473d592e55b6c49e7a0a18a9d53270b59/docker/worker/Dockerfile#L12?

Reply:
I see the current PR as ready to be merged but based on your comment I have also provided https://github.com/os-autoinst/os-autoinst/pull/1573 now with an alternative approach. They don't conflict and we can also change Dockerfile.qemu-x86 later if the other packages are provided.

Reply:
Thanks, I think I'll approve both :+1: